package.nls.json

```diff
@@ -37,5 +37,6 @@
     "configuration.maven.terminal.favorites.alias": "A short name for the command.",
     "configuration.maven.terminal.favorites.command": "Content of the favorite command.",
     "configuration.maven.terminal.favorites.debug": "Whether to execute in debug mode.",
-    "configuration.maven.settingsFile": "Specifies the absolute path of your maven configuration file, the default value is ~/.m2/settings.xml"
+    "configuration.maven.settingsFile": "Specifies the absolute path of your maven configuration file, the default value is ~/.m2/settings.xml",
+    "configuration.maven.warnConflictsInDiagnostics": "Specify whether to show conflict dependencies in pom.xml files or Problems channel."
```

Comment:
Better to refine the wording.

> in pom.xml files or Problems channel.

"Problems channel" is not accurate, and I think the point is to emphasize:
1) whether to show corresponding dianostics.
2) diagnotics are about dependency conflict.